From: SJ Park <sj@kernel.org>
Date: Thu, 21 Sep 2023 09:41:45 +0000
Subject: [PATCH] Revert "kselftest/runner.sh: Propagate SIGTERM to runner
 child"

This reverts commit 9616cb34b08ec86642b162eae75c5a7ca8debe3c.

The commit makes 'stty' hungup, which is used by kunit in
damon-tests/corr.  Revert the commit as a temporal workaround for now.

Signed-off-by: SJ Park <sj@kernel.org>
---
 tools/testing/selftests/kselftest/runner.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/testing/selftests/kselftest/runner.sh b/tools/testing/selftests/kselftest/runner.sh
index 45fb8ec38493..991207d34bb1 100644
--- a/tools/testing/selftests/kselftest/runner.sh
+++ b/tools/testing/selftests/kselftest/runner.sh
@@ -50,7 +50,7 @@ tap_timeout()
 	# Make sure tests will time out if utility is available.
 	elif [ -x /usr/bin/timeout ]; then
 		/usr/bin/timeout --foreground "$kselftest_timeout" \
-			/usr/bin/timeout "$kselftest_timeout" $1
+			$1
 	else
 		$1
 	fi
-- 
2.47.3

